matplotlib/src/MatPlotLibPanel.tsx

```diff
@@ -36,7 +35,7 @@ export type MatPlotLibPanelState = {
 /**
  * Displays a rendered matplotlib from the server
  */
-export const MatPlotLibPanel = (props: MatPlotLibPanelProps): JSX.Element => {
+export function MatPlotLibPanel(props: MatPlotLibPanelProps): JSX.Element {
```

Comment:
May as well be `React.ReactNode` to be consistent
```suggestion
export function MatPlotLibPanel(props: MatPlotLibPanelProps): React.ReactNode {
```